src/Stratis.Bitcoin/Base/TimeSyncBehavior.cs

```diff
@@ -297,6 +293,28 @@ private void RecalculateTimeOffsetLocked()
             this.logger.LogTrace("(-):{0}={1}", nameof(this.timeOffset), this.timeOffset.TotalSeconds);
         }
 
+        /// <summary>
+        /// Gets the median offset of all the inbound samples (limited to 1 less than the OutboundToInboundWeightRatio per outbound) and the outbound samples with a weighting of the OutboundToInboundWeightRatio.
```

Comment:
Split this comment into 2 or 3 lines and use hmldoc-see for the `OutboundToInboundWeightRatio` constant